src/sinks/vector/config.rs

```diff
@@ -35,7 +35,7 @@ pub struct VectorConfig {
     /// Version of the configuration.
     version: Option<super::VectorConfigVersion>,
 
-    /// The downstream Vector address to connect to.
+    /// The downstream address to which to connect.
```

Comment:
This feels like it shouldn't be changed, since the destination is either a Vector instance, or a server accepting Vector protobuf encoded messages.

Reply:
Hey @spencergilbert, I wasn't sure about this either. What about using "instance" then? Or is that still too vague? 
"The downstream instance address to which to connect."

Reply:
🤔 actually looking at a [comparable comment](https://vector.dev/docs/reference/configuration/sinks/prometheus_remote_write/#endpoint) we don't name the service/instance there either. The `prometheus_remote_write` sink would support writing to anything that is compatible with that protocol which includes prometheus, thanos, victoria metrics, vector... `vector` sink is the same, although I know of no third party application that has implemented our protocol 😄 

On second thought I'd say your update is fine, but I would note that the `vector` name is still going to be in the documentation "here" just by nature of the name of the component - which seems fine IMHO as it's akin to the use of `logstash`, `fluent`, `elasticsearch`, etc.

Reply:
@spencergilbert Ohh this is the Vector sink. 🤦‍♀️ Yes, it does make sense that Vector is mentioned here. I'll add it back!